.github/workflows/terraform_pr.yml

```
@@ -0,0 +1,45 @@
+---
+name: Terraform Module
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - edited
+      - synchronize
+jobs:
+  terraform:
+    name: Terraform Format
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Terraform
+        uses: hashicorp/setup-terraform@v1
+        with:
+          terraform_version: 0.14
```

Should this be 
```suggestion
          terraform_version: 1.4.5
```
to match `.terraform-version`?